frontend/src/component/personalDashboard/PersonalDashboard.tsx

```diff
@@ -136,13 +138,56 @@ const useProjects = () => {
     return { projects, activeProject, setActiveProject };
 };
 
+const FlagListItem: FC<{
+    flag: { name: string; project: string; type: string };
+    selected: boolean;
+    onClick: () => void;
+}> = ({ flag, selected, onClick }) => {
+    const IconComponent = getFeatureTypeIcons(flag.type);
+    return (
+        <ListItem key={flag.name} disablePadding={true} sx={{ mb: 1 }}>
+            <ListItemButton
+                sx={projectStyle}
+                selected={selected}
+                onClick={onClick}
+            >
+                <ProjectBox>
+                    <IconComponent color='primary' />
+                    <StyledCardTitle>{flag.name}</StyledCardTitle>
+                    <IconButton
+                        component={Link}
+                        href={`projects/${flag.project}/features/${flag.name}`}
+                        size='small'
+                        sx={{ ml: 'auto' }}
+                    >
+                        <LinkIcon
+                            titleAccess={`projects/${flag.project}/features/${flag.name}`}
+                        />
+                    </IconButton>
+                </ProjectBox>
+            </ListItemButton>
+        </ListItem>
+    );
+};
+
 export const PersonalDashboard = () => {
     const { user } = useAuthUser();
 
     const name = user?.name;
 
     const { projects, activeProject, setActiveProject } = useProjects();
 
+    const { personalDashboard } = usePersonalDashboard();
```

Comment:
I will extract this code into custom hook once we find shared items between flags and projects